src/presentation/api/controller/files.go

```diff
@@ -107,12 +121,34 @@ func (controller *FilesController) Create(c echo.Context) error {
 		}
 	}
 
-	createDto := dto.NewCreateUnixFile(filePath, &filePermissions, fileType)
+	operatorAccountId := service.LocalOperatorAccountId
+	if requestBody["operatorAccountId"] != nil {
+		operatorAccountId, err = valueObject.NewAccountId(
+			requestBody["operatorAccountId"],
```

Comment:
Wait, what? Are you allowing the user to send the operatorAccountId? Cause that would be a magnificent security flaw.